modules/bridge/src/main/ts/ephox/bridge/components/menu/Container.ts

```diff
@@ -0,0 +1,37 @@
+import { FieldSchema, ValueSchema } from '@ephox/boulder';
+import { Result } from '@ephox/katamari';
+import { CardItem } from '../../api/Menu';
+import { autocompleteSchema } from './Autocomplete';
+import { CardItemsSpec } from './CardMenuItem';
+import { descriptionSchema } from './Description';
+import { imageSchema } from './Image';
+
+export interface ContainerSpec {
+  type: 'container';
+  direction?: 'vertical' | 'horizontal';
+  items: CardItemsSpec[];
+}
+
+export interface Container {
+  type: 'container';
+  direction: 'vertical' | 'horizontal';
+  items: CardItem[];
```

Comment:
As discussed in the call, we should possibly rename this to `ContainerItem` or something more generic.

Reply:
I renamed the type to `ContainerItem` earlier but (after some thinking) I can't see how it should be called container. In the case of [Panel](https://github.com/tinymce/tinymce/blob/9f1f378779b2a68d8a4b577f341461616d408ac3/modules/bridge/src/main/ts/ephox/bridge/components/dialog/Panel.ts#L35), the items-type is called `BodyComponent` (which may be a Label or Checkbox etc.).

I changed this again and restructured to reflect the structure of `BodyComponent`. See this [commit](https://github.com/tinymce/tinymce/pull/6149/commits/cd665466a76e7d6fc5b916e7919d656815504aba). Do you think it makes more sense now after the other changes we have made, and how it has become more `Card`-specific overall? if not, I can revert this commit.